### DISCUSSION TITLE
What Challenges have you faced in Research?

### DISCUSSION THREAD

**Discussion by ADParedes** [Open-Research-Program/HCOE#12] [General]
We all come from different backgrounds and experiences.  Let us know what challenges you faced in research and how you overcame them OR how we can support you!

**Comment by ADParedes** [comment]
Personally, I have faced several prejudices in academia from faculty, peers, and even students I have mentored.   I came into my Ph.D. program very young, at 22 and joined a lab where everyone except one was from a different county, including the PI.  Being the youngest and in an unsupportive "don't ask questions" environment, my confidence quickly dropped.  Some labmates (other PhD students) were as much as 10 years older than me, and they were sometimes the opposite of supportive.  I quickly submitted to the culture that was created, which was to not ask questions, assuming eventually I'll get direct mentorship and "I just have to earn it. "  **This was a huge mistake!**  By not freely asking questions about what I wanted and submitting to any direction given, I constantly struggled to grasp the concept of what I needed to do to graduate and found myself making little to no progress in my first 2 years.  
I overcame this trap of assuming "this is just how it was" by seeking advice outside of my lab from other PIs.  I learned that in higher education to be successful I needed to not be afraid to ask questions and stand up for what I needed.  I left my lab and joined another after 3 years.  I started from scratch and completed several projects with publications after that. 
As a young latin Ph.D. student,  I constantly felt like I didn't belong in higher education and that I had to prove myself.  All it made me do was pretend I knew what was going on.  The best thing I learned from this early experience was simple but hard to execute in my early years of research:  Don't be afraid to ask others how they can help you reach your goals.

**Reply by jasson907** [reply]
Honestly, I feel the same. When I finally joined a professional research lab, it was difficult to be taught many things because everyone was so busy so it was either dont ask questions or dont ask "stupid" questions that would "waste" their times (even though the answer would sometimes be less than 2 minutes.) Eventually, due to just being frustrated about not understanding what was going on in the lab half the time, I began to ask as many questions as I needed to. From my personal experience, I got a mixed bag of results. Some were welcoming and acknowledged that they had not explained topics properly, while others just look at me in disbelief as if I had asked some outlandish question. However, in the end, by asking questions, I got much further in the lab than when I was just trusting that I would one day just "get it".

**Comment by spulido1** [comment]
One challenge I have faced in research is email confusion. Often I receive unclear directions. Instead of re-reading the email multiple times and making assumptions, I reply with clarification questions. This is very important in order to avoid making mistakes that could delay the project. Depending on the PI/resident Drs, it’s often even better and quicker to send a question via text or phone call. At first I felt like I was being annoying/revealing my ignorance but to this day I do not regret asking any question that I could not answer myself. Also, I think it is appreciated when you communicate that you do not understand/need help.

**Comment by Jdlopez0108** [comment]
One of the most notable challenges I experience in research is the feeling that I don't know enough. I often have anxiety about asserting my ideas for fear that I will say something that I can't explain or support enough. It's intimidating to start helping in a lab with incredibly well-read and experienced researchers. I think forming a relationship and talking to mentors and lab-mates is ultimately what helps to ease this challenge. You also just have to do the work of research to start building an intellectual foundation from which to ask questions. At the end of the day, I think being vocal with your thoughts and ideas creates learning moments for yourself and others and often prompts further inquiry. Even when you're wrong, the act of vocalizing and talking about something you were wrong about should clarify the concept and help you to remember that thing in the future.

**Comment by CharliePerezSuarez** [comment]
I feel like one difficulty I've faced in research is that once I begin working with a new team, like in this summer's research,I always feel like I don't belong because everyone else already knows certain procedures. I also sometimes feel like I'll be getting in the way, since they have to train a whole new person. I do feel like it has gotten better now that I know what to do and more about what the projects are about, but it definitely was difficult at first to fit in.
